### ISSUE TITLE
update nekoray

### ISSUE THREAD

**Issue by iamfarhad** [MatsuriDayo/nekoray#1299]
hi

why you dont update this project?

**Comment by lelik107** [comment]
Yes, dear developers, update Nekoray, please. It has custom build XRay and Sing-Box cores so the end-users can't just update them from the original repositories. And not every user can build the software him/herself.

**Comment by rdbox** [comment]
Is the project even alive? If not, will someone make a fork and bring it up to date? I really like this program, it's super!

**Comment by papampi** [comment]
Check this one: https://github.com/Mahdi-zarei/nekoray

**Comment by rdbox** [comment]
> Check this one: https://github.com/Mahdi-zarei/nekoray

super Thank you! to you kind man)
